class.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+function getInput() {
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  const book = new Book(title.value, author.value);
+  return book;
+}
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    if (getInput().title !== '' && getInput().author !== '') {
+      this.data.push(book);
+      localStorage.setItem('library', JSON.stringify(this.data));
+      addToUI(book);
+    }
+  }
+
+  removeBook(id) {
+    const book = document.getElementById(id);
+    book.remove();
+    this.data = this.data.filter((bookObj) => bookObj.id !== id);
+    localStorage.setItem('library', JSON.stringify(this.data));
+  }
+}
```

Comment:
- [ ] [OPTIONAL] Your code would be much cleaner and easier to maintain if you would have each class in their own separate files 👍